oak_functions_launcher/README.md

```diff
@@ -1,29 +1,29 @@
 # Oak Functions Launcher
 
-Untrusted companion app that loads the Oak Functions binary in either QEMU or
-crosvm, and exposes a gRPC server for communicating with the binary.
-Communicates with the binary via the `oak_channel`.
+The Oak Functions Launcher loads the Oak Functions enclave binary and exposes a
+gRPC server for communicating with the binary via the
+[Oak Channel](../oak_channel).
 
-## Starting the binary
+## Launching the Oak Functions enclave binary
 
-The Oak Functions binary may be loaded in crosvm via
+First, set the permission to access the KVM kernel module by either (i)
+`sudo setfacl -m u:${USER}:rw /dev/kvm` on the host, or by (ii) adding the user
+to the KVM group.
+
+Then, run the following from within the
+[Oak Developer Docker Container](../docs/development.md#docker-helper-scripts):
+
+To launch the Oak Functions enclave binary with `xtask`:
 
 ```shell
-(cd oak_functions_enclave && cargo build) \
-    && RUST_LOG=debug cargo run \
-    --package=oak_functions_launcher \
-    -- \
-    --wasm=oak_functions_launcher/key_value_lookup.wasm \
-    --lookup-data=oak_functions_launcher/mock_lookup_data \
-    crosvm \
-    --vmm-binary=/usr/local/cargo/bin/crosvm \
-    --enclave-binary=oak_functions_enclave/target/x86_64-unknown-none/debug/oak_functions_enclave
+xtask run-launcher-test
 ```
 
-See also the See the task integration at `xtask/src/vm.rs`. Additional
-documentation is available via
+Additional documentation is available via
 `cargo run --package=oak_functions_launcher -- --help`.
 
+To launch the Oak Functions enclave binary directly as a child process:
```

Comment:
This is the Oak Functions Linux binary, not the enclave binary.

Reply:
Thank you!